test/Microsoft.TestPlatform.AcceptanceTests/Extension/RunnnerInfo.cs

```diff
@@ -1,84 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#nullable disable
-
 using System;
 
 namespace Microsoft.TestPlatform.AcceptanceTests;
 
-public class RunnerInfo
+/// <summary>
+///
+/// </summary>
+/// <param name="RunnerFramework"></param>
+/// <param name="TargetFramework"></param>
+/// <param name="InIsolationValue">Supported value = <c>/InIsolation</c>.</param>
+public record RunnerInfo(string RunnerFramework, string TargetFramework, string InIsolationValue = "")
 {
-    public RunnerInfo(string runnerType, string targetFramework) : this(runnerType, targetFramework, "")
-    {
-    }
-
-    public RunnerInfo(string runnerType, string targetFramework, string inIsolation)
-    {
-        RunnerFramework = runnerType;
-        TargetFramework = targetFramework;
-        InIsolationValue = inIsolation;
-        // The value is netcoreapp2.1.
-        IsNetRunner = RunnerFramework.StartsWith("netcoreapp", StringComparison.InvariantCultureIgnoreCase);
-        // The value is net451.
-        IsNetFrameworkRunner = !IsNetRunner;
-        IsNetTarget = TargetFramework.StartsWith("netcoreapp", StringComparison.InvariantCultureIgnoreCase);
-        IsNetFrameworkTarget = !IsNetTarget;
-    }
-
-    /// <summary>
-    /// Gets the target framework.
-    /// </summary>
-    public string TargetFramework
-    {
-        get;
-        set;
-    }
-
-    /// <summary>
-    /// Gets the inIsolation.
-    /// Supported values = <c>/InIsolation</c>.
-    /// </summary>
-    public string InIsolationValue
-    {
-        get; set;
-    }
-
-    /// <summary>
-    /// Gets the application type.
-    /// </summary>
-    public string RunnerFramework
-    {
-        get;
-        set;
-    }
-
-    public override string ToString()
-    {
-        return string.Join(",", new[] { "RunnerFramework = " + RunnerFramework, " TargetFramework = " + TargetFramework, string.IsNullOrEmpty(InIsolationValue) ? " InProcess" : " InIsolation" });
-    }
-
     /// <summary>
     /// Is running via .NET "Core" vstest.console?
     /// </summary>
-    /// <returns></returns>
-    public bool IsNetRunner { get; }
+    public bool IsNetRunner => RunnerFramework.StartsWith("netcoreapp", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
if you are making changes maybe change this to be on the IsNetFramework runner and say StartsWith "net4"?